activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -425,6 +425,7 @@ def column_name_for_operation(operation, node) # :nodoc:
       def get_database_version
         @connection.server_version
       end
+      alias_method :postgresql_version, :get_database_version
```

Comment:
Use `database_version`.

```suggestion
      alias :postgresql_version :database_version
```

Reply:
done